docs/install.rst

```diff
@@ -52,6 +52,39 @@ so that you can do changes and submit patches.
 [Linux/macOS] Install via your package manager
```

Comment:
I'd rather remove this section altogether. Installing tox not in a virtual environment via pipx is always causing just problems because OS distro repackaging always messes up our packaging.

Reply:
+1

Reply:
While I agree that packaged versions of tox, which are usually outdated and missing bugfixes, will cause some problems, both for the user and for us when the users complain about long fixed bugs or even things which are only broken in the packaged version (looking at you, bionic's version of tox), I do think many people will still use the packaged versions, maybe they even have no choice, e.g. in an enterprise environment.

I am slightly in favor to accept this PR, as it will help especially beginners to get started.

Though, I would like to see some words of warning that the preferred way for installation would be pip/pipx, and that packaged versions of tox might be outdated and are not directly supported by us.

Can we agree on one solution to move forward this pr? Though it looks like a clear 2-1 :-)